src/styles/globals.css

```
@@ -21,17 +21,28 @@ body {
+  from {
+    width: 0;
+  }
+  to {
+    width: 10ch;
+  }
 }
 
 @keyframes blink-caret {
-  from, to { border-color: transparent; }
-  50% { border-color: var(--color-accent); }
+  from,
```

Stylelint (17.13.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)